demo/local-device/readme.md

```diff
@@ -0,0 +1,131 @@
+### Prerequisite
+OpenEBS must be installed.
+### Install cspc-operator
+To provision a CSPC using dao you need to install  [cspc-operator](https://docs.openebs.io/docs/next/alphafeatures.html#install-openebs-cspc-operator) from alpha feature.
+### Install cstorpoolauto DAO operator
+Sample cstorpoolauto DAO operator yaml is given below.
+
+NOTE:- you can install it in any namespace.
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: maya-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cstorpoolauto
+  namespace: maya-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cstorpoolauto
+rules:
+- apiGroups:
+  - "*"
+  resources:
+  - cstorclusterconfigs
+  - blockdevices
+  - cstorpoolclusters
+  - nodes
+  verbs:
```

Comment:
can we provide appropriate verbs per resources?
Atleast mention the same as comments. 
We can come back with granular control in a later PR